ls-ui-react-app/src/components/LatticeView.tsx

```diff
@@ -16,15 +58,41 @@ type CellViewerProps = {
     col_idx: number
 }
 const CellViewer = ({cell, row_idx, col_idx}: CellViewerProps) => {
+    // console.log(cell)
     return <div
             className="lattice-cell-inside"
-            css={css`height: 50pt;
-                  width: 50pt;
-                  vertical-align: middle;
-                  display: inline-block;
-                  border-width: 4pt;
-                  border-style: solid;
-                  text-align: center;`}>
+            css={css`
```

Comment:
One thing that seems to be missing is the gridlines. I couldn't find where to restore that.